jetbrains-gateway/README.md

```diff
@@ -41,18 +41,18 @@ module "jetbrains_gateway" {
 }
 ```
 
-### Use the latest release version
+### Use the fixed set by `jetbrains_ide_versions`
```

Comment:
Customers won't know what this means.
Either we should say the below (since we're _interpreting_ the stability of a release by setting it as our default), or we should link to the definition.

```suggestion
### Use the latest stable release
```

Reply:
As the default behavior is now changed so I wanted to change the example to teach someone on how to use the fixed version passed by `jetbrains_ide_versions`.

Reply:
What is your suggestion on how to improve it? We can add a detailed example on using custom versions using the `jetbrains_ide_versions` varibale.

Reply:
I gave two options here: https://github.com/coder/modules/pull/355#discussion_r1860562310

I think we should allow the user to specify their own `build_number` and `version`, TBH.
If I'm an operator and I know that a certain version is stable, I won't want coder messing that up by always using latest.

Reply:
Yes, then I should just set `stable = false` which stop Coder from messing it up.

Reply:
> I think we should allow the user to specify their own build_number and version, TBH.

This is tricky as diffrent JetBrians IDEs share the same `version` but different in `build_number`s.